### ISSUE TITLE
Tests fail (with Dist::Zilla 6?)

### ISSUE THREAD

**Issue by eserte** [garfieldnate/Dist-Zilla-PluginBundle-NGLENN#3]
My smokers started to report the following test failure:

```
[@NGLENN/NextRelease] failed to find Changes in the distribution at /opt/perl-5.25.1t/lib/site_perl/5.25.1/x86_64-linux-thread-multi/Moose/Meta/Method/Delegation.pm line 110.
t/nglenn.t ............. 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
```

Statistical analysis suggests that the problem could be caused by Dist::Zilla 6:

```
****************************************************************
Regression 'mod:Dist::Zilla'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    139133882289116784.00
[1='eq_5.039']        0.0000          0.0000       0.60
[2='eq_5.041']        0.0000          0.0000       0.75
[3='eq_5.042']        0.0000          0.0000       0.00
[4='eq_5.043']        0.0000          0.0000       1.20
[5='eq_5.044']        0.0000          0.0000       0.00
[6='eq_5.047']        0.0000          0.0000       3.24
[7='eq_6.005']       -1.0000          0.0000    -58447921314884296.00

R^2= 1.000, N= 57, K= 8
****************************************************************
```

**Comment by garfieldnate** [comment]
Thanks, I will look into this.

**Comment by garfieldnate** [comment]
Fixed in b37cbaf07440cb.
